cube/src/main/java/zone/cogni/asquare/cube/convertor/data2shacl/Configuration.java

```diff
@@ -61,10 +69,29 @@ public void setReportPossibleIssues(boolean reportPossibleIssues) {
   }
 
   public List<String> getIgnoredClasses() {
-    return ignoredClasses;
+    return ListUtils.emptyIfNull(ignoredClasses);
   }
 
   public void setIgnoredClasses(List<String> ignoredClasses) {
     this.ignoredClasses = ignoredClasses;
   }
+
+  public Map<Set<String>, String> getTypeTranslations() {
+    return MapUtils.emptyIfNull(typeTranslations);
+  }
+
+  public void setTypeTranslations(Map<Set<String>, String> typeTranslations) {
+    this.typeTranslations = typeTranslations;
+  }
+
+  public String getTypeTranslation(Set<String> types) {
+    if (typeTranslations == null) return null;
+    return typeTranslations.get(types);
+  }
+
+  public void addTypeTranslation(Set<String> types, String type) {
+    if (typeTranslations == null) typeTranslations = new HashMap<>();
```

Comment:
might be good to handle this in initialization in order to get rid of such checks